### PR TITLE
Review of starter app

### PR DESCRIPTION
**Pull request by jenperson**
Minor tweaks to make the app more Squid-y and to add directions to README

### CODE REVIEW

**Review by tsouza-squid — APPROVED** [review]

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -16,31 +15,22 @@ function App() {
...
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
+        <a href='https://docs.squid.cloud' target='_blank'>
+          <img src={squidLogo} className='logo' alt='Squid Cloud logo' />
         </a>
       </div>
       <h1>Vite + React</h1>
-      <div className="card">
+      <div className='card'>
```
Comment (ohsnapitscolin) [inline frontend/src/App.tsx]: I think we might want to install `prettier` in all of these repos (`npm install -D prettier`) can configure it in your IDE to run on save. Just will ensure that everything is consistent (prettier will make this double quotes).

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -16,31 +15,22 @@ function App() {
   return (
```
Comment (ohsnapitscolin) [inline frontend/src/App.tsx:18]: Just a note that the `subscribe: true` above is not required, it's the default.

File: frontend/src/App.tsx [hunk 3]
```diff
@@ -16,31 +15,22 @@ function App() {
...
-        </p>
       </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
     </>
-  )
+  );
 
   async function handleClick() {
```
Comment (ohsnapitscolin) [inline frontend/src/App.tsx]: I would move this above the return statement, typically the return comes at the end of the function.

File: frontend/src/App.tsx [hunk 4]
```diff
@@ -16,31 +15,22 @@ function App() {
...
     await collection.doc('count').insert({ count: (data?.count || 0) + 1 });
```
Comment (ohsnapitscolin) [inline frontend/src/App.tsx]: I might break this into two lines for clarity:
```
const count = (data?.count || 0) + 1;
await collection.doc('count').insert({ count });
```

File: frontend/public/Squid-Symbol-Dark.svg
Comment (ohsnapitscolin) [inline frontend/public/Squid-Symbol-Dark.svg:1]: ultra nit: would probably just rename this to `squid.svg`

**Review by ohsnapitscolin — APPROVED** [review]
💯

**Pull request merged**